Legende dos status:

| Status | Significado |
| :-: | :- |
| ✅ | Rodou tudo certinho |
| 🟧 | Rodou parcialmente |
| ❌ | Não rodou |
| 🕐 | Timeout |

**Data dos testes:** Sep 28 / 16:18
| name | status | log |
| :-: | :-:| :- |
| guilhermedjr | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| brunokim | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| iaZe | 🟧 | luajit: interpreter.lua:134: attempt to index local 'tuple' (a nil value) stack traceback: 	interpreter.lua:134: in function 'interpreter' 	interpreter.lua:133: in function 'interpreter' 	interpreter.... | 
| joselsneto | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| Emanuel Júnior | ❌ | /app/utils.py:93: RuntimeWarning: overflow encountered in scalar add   "Add": lambda x, y: str(x) + str(y) if type(x) == str or type(y) == str else x + y, Traceback (most recent call last):   File "/a... | 
| rwillians | ❌ | cannot clone repository 'authentication required but no callback set' | 
| eduhenke | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| Marcelo Fernandes | ✅ | Exception in thread "main" java.lang.ArithmeticException: long overflow 	at java.base/java.lang.Math.multiplyExact(Math.java:1004) 	at clojure.lang.Numbers.multiply(Numbers.java:1971) 	at clojure.lang... | 
| Joshua Passos | ❌ | The command '/bin/sh -c stack build' returned a non-zero code: 1 | 
| ThiagoTatico | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| JrSchmidtt | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| jhonnk | ❌ | Hello world  | 
| ErnaneJ | ❌ | ( 5 ,  8 )  | 
| Rodrigo Meray | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| lucasmontano | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| joaomarcosbc | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| [João Paulo](https://github.com/joaopaulosg21/) | ❌ | Exception in thread "main" aprendendo.exceptions.ParamsException: Quantidade de parametros e argumentos diferentes ... | 
| Diego Ramos | ❌ | Traceback (most recent call last):   File "/app/src/python/entrypoint.py", line 16, in <module>     interpret(filename, f.read())   File "/app/src/python/entrypoint.py", line 10, in interpret     ast ... | 
| Edmar Sousa | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| edusporto | ❌ | cannot clone repository 'authentication required but no callback set' | 
| rcovery (Ryan) | ✅ | ** (MatchError) no match of right hand side value: {:error, :enoent}     (rinha 0.1.0) lib/error.ex:6: Error.handle/2     (rinha 0.1.0) lib/interpreter.ex:120: Interpreter.do_binary/2     (rinha 0.1.0... | 
| Ronaldo3030 | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Leandro Proença | 🟧 | /app/lib/trampoline.rb:24:in `rescue in block in run': Error: Unexpected error while evaluating continuation: Error: Cannot call ((1, 1), ) with (2, ): undefined method `call' for [[1, 1], nil]:Array ... | 
| Wanderson Ralph | 🟧 |  | 
| BertBR | ❌ | var fib func(n) { if n < 2 { var n } else {  fib(n - 2)  | 
| Pedro Nogueira | ✅ | $ bun run --target="bun" dist/index.js  | 
| Daniel Wojcickoski | ✅ | ╭───────────────────── Traceback (most recent call last) ──────────────────────╮ │ /usr/local/lib/python3.11/site-packages/rinha_interpreter/main.py:33 in main │ │                                     ... | 
| mtrecouso | ❌ | /var/app/optimize.js:102       functionCalls[scope].isPure = false;                                   ^  TypeError: Cannot set properties of undefined (setting 'isPure')     at optimizeRecursions (/va... | 
| migfernandes01 | ❌ | panic: runtime error: index out of range [1] with length 1  goroutine 1 [running]: main.main() 	/main.go:23 +0x1fc ... | 
| FlorSousa | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Yuri Withowsky | ❌ | Unhandled exception. System.Text.Json.JsonReaderException: The maximum configured depth of 64 has been exceeded. Cannot read next JSON object. LineNumber: 0 | BytePositionInLine: 95130.    at System.T... | 
| Patrick | ❌ | 55  | 
| PedroFnseca | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| cleissonbarbosa | ✅ | /app/run.sh: line 2:     6 Killed                  julia main.jl --file=/var/rinha/source.rinha.json ... | 
| Aleff Moura | ❌ | Unhandled exception. System.IndexOutOfRangeException: Index was outside the bounds of the array.    at Program.main(String[] args) in /src/Program.fs:line 47 ... | 
| mr-soulfox | ❌ |  | 
| danielHc91 | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| lucaspsg | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| dhilst | ❌ |  | 
| Gabriel Martins | ❌ | 1 | import { Expression } from "./interfaces"; 2 |  3 | export function interpretExpression(expression: Expression, environment: Record<string, any>): any { 4 |     switch (expression.kind) {         ... | 
| matheusgb | ❌ | error 400 Bad Request - the Dockerfile (Dockerfile) cannot be empty | 
| Matheus Lenke | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| DiegoTeles | ❌ | COPY failed: no source files were specified | 
| Ytu | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| David Lins | ❌ | Exception in thread "main" java.lang.IllegalArgumentException: Unexpected kind value: If 	at br.com.rinha.compiladores.translate.RinhaAstToJavaSource.handleExpression(RinhaAstToJavaSource.java:272) 	a... | 
| eduOliver | ❌ |  > rinha-compiler@1.0.0 start > ts-node src/main.ts  /usr/src/app/files/combination.json /usr/src/app/files/combination.rinha /usr/src/app/files/fib.json /usr/src/app/files/fib.rinha /usr/src/app/file... | 
| Leonardo Di Credico e Gabriel Lima | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Luiz Antônio | ❌ | /usr/local/bin/herbiec.ksh[466]: main[43]: interpreter[58]: json2compound[394]: eval: syntax error at line 1: `((' unexpected ... | 
| luizzuchinali | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| italo | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| Breno Carvalho | 🟧 | ERROR: Type error: `number' expected, found `_1348520{kind:"Function",location:_1348678{end:18424,filename:"tests/source.rinha",start:18409},parameters:[_1348650{location:_1348662{end:18414,filename:"... | 
| erik guizao | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| samueldurantes | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Victor Augusto | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| wilgnne | ❌ |  <--- Last few GCs --->  [1:0x70293c0]    32404 ms: Mark-sweep 990.5 (1043.6) -> 985.9 (1047.3) MB, 66.3 / 0.0 ms  (average mu = 0.939, current mu = 0.494) allocation failure; scavenge might not succe... | 
| fenner | ❌ | A execução do aplicativo falhou Final do código não esperado. Posição (1870). Linha(95). Coluna(37)  | 
| astahjmo e niumxp | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| celsobonutti | ❌ | uncaught exception: occurs check fails  | 
| rodrigocam | ❌ | thread 'main' panicked at 'index out of bounds: the len is 1 but the index is 1', src/main.rs:11:35 note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace ... | 
| Samuel Haidu | ❌ | cannot clone repository 'unexpected http status code: 404' | 
| datsfilipe | ✅ | ENOENT: No such file or directory    path: "tests/source.rinha"  syscall: "open"    errno: -2  ENOENT: No such file or directory    path: "tests/source.rinha"  syscall: "open"    errno: -2  ENOENT: No... | 
| fernandozanutto | 🟧 | Traceback (most recent call last):   File "//main.py", line 22, in <module>     main()   File "//main.py", line 18, in main     Interpreter(decoded_json).run()   File "/interpreter.py", line 263, in r... | 
| Mikhail R. Gadelha | ❌ | generated_main.cpp: In function 'auto map(T0, T1)': generated_main.cpp:11:48: error: 'nil' was not declared in this scope    11 | return cons(new_val, new_rest); } else {return nil;}}template <typenam... | 
| Rafa Dutra | ❌ | /usr/local/bundle/gems/json-2.6.3/lib/json/common.rb:216:in `parse': nesting of 101 is too deep (JSON::NestingError) 	from /usr/local/bundle/gems/json-2.6.3/lib/json/common.rb:216:in `parse' 	from src... | 
| Felipe Cardoso | 🟧 | file:///interpreter/interpreter.js:104       throw new Error(`Unknown node kind: ${node.kind}`);             ^  Error: Unknown node kind: First     at interpret (file:///interpreter/interpreter.js:104... | 
| Maurício Mussatto Scopel | 🟧 | line 26:18 mismatched input '/' expecting {'(', '+', '-', 'let', 'fn', 'if', 'print', 'first', 'second', INT, BOOL, ID, STRING} line 26:30 mismatched input 'is' expecting {'(', ')', '+', '-', '*', '/'... | 
| oMuriloDev | 🟧 | panic: interface conversion: main.Term is int32, not bool  goroutine 1 [running]: main.Eval(0xc000499dd0, {0x500500?, 0xc0000f5410}) 	/eval.go:121 +0x1dfd main.Eval(0xc000499dd0, {0x500500?, 0xc0000f5... | 
| João Vitor (Jottinha) | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| maxwellmonteiro | ❌ |  | 
| leodhb | ❌ | exec ./script.sh: exec format error  | 
| danielhoffmanno | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| Douglas S. Elias | ❌ | build/program.rb:30: syntax error, unexpected `next' def cons(value,next)                  ^~~~  | 
| guitcastro | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| cleissonbarbosa | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| lucaspacheco | 🟧 | Erro ao executar .rinha.json: Cannot implicitly convert type 'int' to 'bool'  | 
| ghhernandes | 🟧 | panic: tests/source.rinha:286:291: undefined variable value  goroutine 1 [running]: github.com/ghhernandes/rinha-compiler-go/runtime.Error(...) 	/app/rinha-go/runtime/runtime.go:9 github.com/ghhernand... | 
| Crazynds | ❌ | The command '/bin/sh -c npm install' returned a non-zero code: 254 | 
| Tacio | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| thebe111 | ❌ | thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("recursion limit exceeded", line: 1, column: 260424)', src/main.rs:354:41 note: run with `RUST_BACKTRACE=1` environment va... | 
| João Pedro F. Barbosa | ❌ | Error parsing JSON: recursion limit exceeded at line 1 column 260424  | 
| Gabriel Tavares (Modog) | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| Lamadelrae | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Maxsuel Maccari | ❌ | bakeware: Error creating directory /root/.cache/bakeware/.tmp/mNCNCM??: No such file or directory bakeware: Unrecoverable validation error ... | 
| Gabriel Santos | ✅ 🕑 |  | 
| fagundesraphael | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| GCdePaula | ❌ | Error: Error("recursion limit exceeded", line: 1, column: 260424)  | 
| Danfs | ❌ | file:///app/a.js:26             ? let _ = (               ^^^  SyntaxError: Unexpected strict mode reserved word     at ModuleLoader.moduleStrategy (node:internal/modules/esm/translators:118:18)     a... | 
| Alex Silva | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Salomemes | ❌ | $ bun run index.ts 3 | } 4 |  5 | export async function evalOutput(filename: string) { 6 |   const content = await Bun.file(filename).text(); 7 |  8 |   eval(content);       ^ SyntaxError: Unexpected ... | 
| MarcosCostaDev | ❌ | Required argument missing for command: 'Rinha'.   | 
| Raphael M. R. Victal | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| tiagosalem | ❌ | parse error: Exceeds depth limit for parsing at line 1, column 276695 Runtime error: undefined kind  ... | 
| LucasVeloz | 🟧 |  node:internal/event_target:1083   process.nextTick(() => { throw err; });                            ^ TypeError [Error]: Cannot read properties of undefined (reading 'value')     at processNode (/sr... | 
| Valmor Flores | ❌ | Unhandled exception: Exception: Parse Error - Unknown expression kind: Tuple #0      Interpreter.parseExpression (file:///intR64/src/interpreter.dart:71) #1      Interpreter.parseExpression (file:///i... | 
| Paulo Victor | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| lrlucena | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Teles | ❌ | thread 'main' panicked at 'failed to parse ast file: Error("recursion limit exceeded", line: 1, column: 260424)', src/main.rs:14:61 note: run with `RUST_BACKTRACE=1` environment variable to display a ... | 
| Gustavo Mendonça | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| FlavioNevs | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| RRFreitas | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Vitor Vezani | ❌ | 2023/09/28 18:23:21 error in codegen: first expects a tuple  | 
| Paulo Paes | 🟧 | Failed to compile: Maximum call stack size exceeded  | 
 157 |       if(term.value.kind !== 'Tuple') throw new Error('Term is not T... | 
| netodotcom | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| antoniogomes94 | ❌ | var/rinha/source.rinha.json Interpreter Exec. Time: 292 ms  | 
| davisuga | 🟧 🕑 |  | 
| Balmanth | ❌ | /usr/src/rinha/bin/rinha.js:1 (()=>{var e={2560:(e,t,n)=>{"use strict";Object.defineProperty(t,"__esModule",{value:!0}),t.Data=void 0;const o=n(8088);t.Data=class{#e;get data(){if(!this.#e)throw new o... | 
| dgomesma | ❌ | The command '/bin/sh -c make' returned a non-zero code: 2 | 
| DouglasGabr | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| ericklima-ca | 🟧 | runtime: goroutine stack exceeds 1000000000-byte limit runtime: sp=0xc02da00368 stack=[0xc02da00000, 0xc04da00000] fatal error: stack overflow  runtime stack: runtime.throw({0x51728e?, 0x2030050?}) 	/... | 
| Alessandro Lima | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| Adriano dos Santos Fernandes | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| [Jeff Quesado](https://github.com/jeffque) | ❌ | tá no placeholder Tuple tá no placeholder Tuple tá no placeholder First tá no placeholder Second tá no placeholder Second tá no placeholder First tá no placeholder Second tá no placeholder Second tá n... | 
| matheusBBarni | ❌ | error: missing script "Main.bs.js"  | 
| pandrre | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| ferreirawalter | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Mateus Ryan | ❌ | source.rinha(0): error: Invalid file format  | 
| renatoalencar | ❌ | Type Str cant match IntFatal error: exception Rinha__Typedtree.Unify_error  | 
| kurioh | ❌ | thread 'main' panicked at src/main.rs:25:56: called `Result::unwrap()` on an `Err` value: "Expected `Term`:\n\u{1b}[0m    46 | (DynamicListTest) \u{1b}[4m\u{1b}[31m=\u{1b}[0m (STD.closure (let dyn_lis... | 
| Yago Crispim | 🟧 | lua: ./src/interpreter.lua:166: attempt to call a nil value (local 'op') stack traceback: 	./src/interpreter.lua:166: in function <./src/interpreter.lua:152> 	(...tail calls...) 	./src/interpreter.lua... | 
| lbguilherme | ❌ |  | 
| Victor Hogemann | ✅ | Unhandled exception. System.OverflowException: Value was either too large or too small for a Decimal.    at System.Number.ThrowOverflowException(TypeCode type)    at System.Decimal.DecCalc.ScaleResult... | 
| imaqtkatt | ❌ | File: /var/rinha/source.rinha, Syntax error from 26:19 to 26:20  | 
| Ullas (Ulisses Rosa) | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| ch1n3du | ❌ | cannot clone repository 'remote authentication required but no callback set' | 
| Itallo David | 🟧 | Exception in thread "main" dev.itallodavid.rinhalang.errors.UninitializedVariable: The variable was not initialized: x 	at dev.itallodavid.rinhalang.interpreter.DefaultEnvironment.get(DefaultEnvironme... | 
| Guilherme Melo | ❌ | Unhandled exception. Newtonsoft.Json.JsonReaderException: The reader's MaxDepth of 64 has been exceeded. Path 'expression.next.next.next.next.next.next.next.next.next.next.next.next.next.next.next.nex... | 
| autistadev | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Leonardo Monteiro (quatroka) | 🟧 | maximum recursion depth exceeded  | 
| Isaac Nascimento | 🟧 | Error: kind "undefined" não existe Cannot destructure property 'filename' of 'exp.location' as it is undefined.     at (file://tests/source.rinha:544:545)     at (file://tests/source.rinha:534:546)   ... | 
| Ismael Velten | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| [priv](https://github.com/scriptprivate) | ❌ | $VAR1 = {           'text' => undef,           'location' => {                           'end' => 114,                           'filename' => 'source.rinha',                           'start' => 0   ... | 
| milyth | ❌ | The command '/bin/sh -c apt install -y tcc' returned a non-zero code: 100 | 
| fabriciorby | 🟧 | Execution error (NullPointerException) at interpreter.core/parser (core.clj:21). Cannot invoke "clojure.lang.IFn.invoke(Object)" because "ast" is null  Full report at: /tmp/clojure-1267851154881563440... | 
| MarcosCostaDev | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Lorenzobattistela | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Ítalo Paulino (irbp) | ❌ | The command '/bin/sh -c apt-get update &&     apt-get -y install apt-transport-https wget gpg &&     wget -qO- https://dl-ssl.google.com/linux/linux_signing_key.pub | gpg --dearmor -o /usr/share/keyri... | 
| Márcio Saraiva | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| dlopes7 | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| DevDaaniel | ❌ | error 500 Internal Server Error - unexpected error reading Dockerfile: read /var/lib/docker/tmp/docker-builder2312965983/Dockerfile: is a directory... | 
| Vinícius | ❌ | Traceback (most recent call last):   File "/app/main.py", line 14, in <module>     ir.generate(ast_data['expression'])   File "/app/rinha.py", line 44, in generate     self.generate(self.__next_expr.p... | 
| Guilherme Meira | ❌ | Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0 	at org.objectweb.asm.Frame.merge(Frame.java:1269) 	at org.objectweb.asm.Frame.merge(Frame.java:... | 
| Vinicius Ferraz | ❌ | line 1:4 token recognition error at: '_' line 1:6 no viable alternative at input 'let=' line 1:14 token recognition error at: '"@' line 1:16 token recognition error at: '!c' line 1:24 token recognitio... | 
| leonardohn | ❌ | Error: Error("recursion limit exceeded", line: 1, column: 260424)  | 
| breno5g | 🟧 | Error when trying to execute the rinha file:  Unexpected node kind: Tuple  | 
| Erik Morbach | ❌ |  | 
| fernando oliveira | ❌ | no main manifest attribute, in *.jar  | 
| Braayy | ✅ 🕑 |  | 
| Nicolas Van Klaveren | ❌ | Unhandled exception. System.Text.Json.JsonException: The maximum configured depth of 64 has been exceeded. Cannot read next JSON object. Path: $.expression | LineNumber: 0 | BytePositionInLine: 95130.... | 
| Kadu | ❌ | fib: 55 Tempo de execução: 0.00472  | 
| FabioSeixas | ❌ | thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("recursion limit exceeded", line: 1, column: 260424)', src/bin/interpreter.rs:9:54 note: run with `RUST_BACKTRACE=1` envir... | 
| Fabricio Juliatto | 🟧 | Error: Variable rest not found.  | 
| ricardopieper | ❌ |   Failure recursion limit exceeded at line 1 column 260424     | 
| fabriciorby & perrutbruno | ❌ | lua: main.lua:15: Could not find the AST file stack traceback: 	[C]: in function 'error' 	main.lua:15: in function 'main' main.lua:107: in main chunk 	[C]: in ? ... | 
| italotabatinga | ❌ | [line 20] Error at 'nil': Expect variable name. [line 96] Error at 'acc': Expect '{' before function body. [line 104] Error at 'append': Can't read local variable in its own initializer. [line 116] Er... | 
| Gabriel Capanema | ❌ | The command '/bin/sh -c apt update && apt install -y curl unzip' returned a non-zero code: 100 | 
| andrecoelho | ❌ | Unhandled exception. Newtonsoft.Json.JsonReaderException: The reader's MaxDepth of 64 has been exceeded. Path 'expression.next.next.next.next.next.next.next.next.next.next.next.next.next.next.next.nex... | 
| MartinGHaas | ❌ | /app/execute.js:13     throw error;     ^  SyntaxError: Unexpected end of JSON input     at JSON.parse (<anonymous>)     at executeRinha (/app/execute.js:7:22)     at Object.<anonymous> (/app/execute.... | 
| reonardoleis | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Marcelo Magalhães | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| fabiosvm | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| Felipe (Frai) | ❌ 🕑 |  | 
| Enzo Massaki | ❌ | Traceback (most recent call last):   File "/app/main.py", line 113, in main     ast = dict_to_ast(data)   File "/app/main.py", line 97, in dict_to_ast     expression = convert_expression(data["express... | 
| losty17 | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Natã Estanislau | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| Gabriel Mello de Oliveira | 🟧 | panic: interface conversion: interface {} is int32, not map[string]interface {}  goroutine 1 [running]: github.com/gabrielmellooliveira/rinha-de-compiler/src/models.GetField(...) 	/app/src/models/ast.... | 
| lurodrigo | ❌ | ** (FunctionClauseError) no function clause matching in Rinha.to_ir/1              The following arguments were given to Rinha.to_ir/1:              # 1         nil          Attempted function clauses... | 
| Josuel | 🟧 |  Error: Unexpected token ( Token: /, Type: 29, File: /var/rinha/source.rinha, Line: 340, Pos: 1, Stack: 0 )  // fibonacci simple test... | 
| dhrleandro | ❌ |  Fatal error: Uncaught Exception: Error Processing Request in /app/src/Compiler.php:61 Stack trace: #0 /app/src/Compiler.php(220): LeandroDaher\RinhaDeCompilerPhp\Compiler->compile(Object(stdClass), O... | 
| aug | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| BRonen | ❌ | lua: main.lua:14: attempt to call a nil value stack traceback: 	main.lua:14: in main chunk 	[C]: at 0x562d3e807f80 ... | 
| giancarlopro | ❌ |  | 
| lucasnlm | 🟧 | e: invalid sub expression, using v1.2.5  | 
| Jackson Samuel | ❌ | The command '/bin/sh -c npm run build' returned a non-zero code: 2 | 
| Olordecoelho | ❌ | Traceback (most recent call last):   File "/var/rinha/indio.py", line 44, in <module>     print(Interpreter(program_node).run())           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^   File "/var/rinha/src/interp... | 
| nobytes | ❌ | panic: strconv.Atoi: parsing "4.29457395e+08": invalid syntax  goroutine 1 [running]: main.main() 	/hina/cmd/hina/main.go:38 +0x199 ... | 
| Pr3d4dor | ✅ | hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe hehe ... | 
| Rodrigo Navarro | ❌ | thread 'main' panicked at 'Programa inválido: Error("recursion limit exceeded", line: 1, column: 260424)', src/main.rs:368:58 note: run with `RUST_BACKTRACE=1` environment variable to display a backtr... | 
| lrlucena | ❌ | COPY failed: file not found in build context or excluded by .dockerignore: stat rinha-lrlucena.jar: file does not exist... | 
| carlinhos | 🟧 | 39 |       } 40 |     case "Tuple": 41 |       const tuple: any = [evaluate(node.first), evaluate(node.second)]; 42 |       return tuple; 43 |     case "First": 44 |       const t1: any[] = evaluate(n... | 
| Garicas | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| D4yvid | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| cleissonbarbosa | ✅ 🕑 |  | 
| Victor Taelin | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| henri | ❌ | thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error("recursion limit exceeded", line: 1, column: 260424)', src/main.rs:26:57 note: run with `RUST_BACKTRACE=1` environment var... | 
| gabrielogregorio | 🟧 | /app/dist/interpreter.js:65             throw new Error(`First needs use only Tuple`);             ^  Error: First needs use only Tuple     at interpreter (/app/dist/interpreter.js:65:19)     at inter... | 
| Heraldo Lucena | ❌ | error 500 Internal Server Error - Cannot locate specified Dockerfile: Dockerfile | 
| matgomes | ❌ | Get "https://registry-1.docker.io/v2/": net/http: request canceled (Client.Timeout exceeded while awaiting headers)... | 
| mateuxlucax | 🟧 | Unhandled exception: FormatException: Invalid radix-10 number (at character 1) :: ^  #0      int._handleFormatError (dart:core-patch/integers_patch.dart:126) #1      int._parseRadix (dart:core-patch/i... | 
| coproduto | ✅ | thread 1 panic: integer overflow /build/src/main.zig:614:57: 0x2be20a in performArithmeticBinop (rinha_zig) /build/src/main.zig:602:42: 0x2a8891 in evalEagerBinopWithContext (rinha_zig) /build/src/mai... | 
